openreview/arr/webfield/programChairsWebfield.js

```diff
@@ -146,6 +146,193 @@ return {
       })
       return checklistReplies?.length??0;
       `
-    }    
+    },
+    reviewerEmailFuncs: [
+      {
+        label: 'Reviewers with Unreplied Rebuttals', filterFunc: `
+        console.log(row);
+        if (row.notesInfo.length <= 0){
+          return false;
+        }
+
+        return row.notesInfo.some(obj => {
+          const reviewId = obj?.officialReview?.id ?? ''
+          const anonId = obj?.officialReview?.anonymousId ?? ''
+          const replies = obj?.note?.details?.replies ?? []
+          const authorReplies = replies.filter(reply => 
```

Comment:
when getting authorReplies, is the invitation we are looking for those notes just Official_Comment? I am testing this emailing option. It looks like an email would be sent to `reviewer1@aclrollingreview.com` but their review does not have a rebuttal, just a Revie_Rating reply by the authors. I am assuming in this case no email should be sent to them?

Reply:
I agree,  `reviewId` mustn't be empty. If it is empty then don't check it.